llvm/lib/Passes/PassRegistry.def

```diff
@@ -142,6 +143,7 @@ MODULE_PASS("tsan-module", ModuleThreadSanitizerPass())
 MODULE_PASS("verify", VerifierPass())
 MODULE_PASS("view-callgraph", CallGraphViewerPass())
 MODULE_PASS("wholeprogramdevirt", WholeProgramDevirtPass())
+MODULE_PASS("write-bitcode", BitcodeWriterPass(nulls(), true))
```

Comment:
What's the point of including this here? The canonical way to use the bitcode writer pass seems to be to add it manually to the pipeline so whatever is using the pipeline can set it's own output stream (which is just set to null here, seeming to make the pass not really useful?).

Reply:
I'm assuming that error is from running the opt invocation with the textual representation of the pass pipeline from `-print-pipeline-passes`? What is your invocation to get the Bitcode Writer in the printed pass pipeline? That doesn't occur (for me) with ToT opt with something like `opt -passes='default<O3>' -print-pipeline-passes test.ll` for example.

Reply:
Ah. I should've figured `-disable-output` on my invocation would never create the bitcode writer. I'm not convinced the current approach (currently manually calling `addClassToPassName` in the `PassBuilder` constructor). Others probably have better ideas on the best approach to fix this.